nixos/tests/gobgpd.nix

```diff
@@ -0,0 +1,47 @@
+import ./make-test-python.nix ({ pkgs, ... } : {
+  name = "gobgpd";
+
+  meta = with pkgs.lib.maintainers; {
+    maintainers = [ higebu ];
+  };
+
+  nodes = {
+    node = { ... }: {
```

Comment:
The test should likely have multiple nodes to establish bgp sessions amongst each other.